packages/storybook/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
```

Comment:
Should this match the other package versions just because?

Reply:
naaaaaaaah :p thinking this is it's own private package that nothing should depend on and we should never bump